packages/components/pie-modal/src/index.ts

```diff
@@ -350,7 +351,7 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
             ?isFullWidthBelowMid=${isFullWidthBelowMid}
             ?isLoading=${isLoading}
             aria-busy="${isLoading ? 'true' : 'false'}"
-            aria-label="${isLoading ? aria?.loading : nothing}"
+            aria-label="${isLoading ? (aria?.loading || nothing) : nothing}"
```

Comment:
I think this can actually be simplified further to `(isLoading && aria?.loading) || nothing`. Personally I think they can all be a bit tricky to read but hopefully with the shorter one there is less that needs to be broken down 😄

Reply:
yes go ahead 🔥